src/App.js

```diff
@@ -10,6 +10,7 @@ function App() {
         <Route path="/" element={<Notes />}></Route>
         <Route path="/login" element={<Login />}></Route>
         <Route path="/signup" element={<Signup />}></Route>
+        <Route path="/archives" element={<Archives />}></Route>
         <Route path="/mock" element={<Mockman />}></Route>
       </Routes>
```

Comment:
You can create a separate routes folder to add the routes.

Reply:
Yess I will refactor that